e2e-tests/src/test/fee.rs

```diff
@@ -113,26 +109,17 @@ fn assert_no_scaling(
     );
 }
 
-fn prepare_transaction(connection: &SignedConnection) -> TransferTransaction {
+fn prepare_transaction<C: BalanceTransfer>(connection: &C) -> C::TransferTx {
     let bytes = [0u8; 32];
-    compose_extrinsic!(
-        connection.as_connection(),
-        "Balances",
-        "transfer",
-        GenericAddress::Id(AccountId::from(bytes)),
-        Compact(0u128)
-    )
+    let account = AccountId::from(bytes);
+
+    connection.create_transfer_tx(account, 0u128.into())
 }
 
 fn fill_blocks(target_ratio: u32, blocks: u32, connection: &RootConnection) {
     for _ in 0..blocks {
-        let call = compose_call!(
```

Comment:
This method we can extract to `aleph-client`  to `system` module https://github.com/Cardinal-Cryptography/aleph-node/blob/main/aleph-client/src/system.rs